packages/instant/tsconfig.json

```diff
@@ -2,16 +2,11 @@
     "extends": "../../tsconfig",
     "compilerOptions": {
         "outDir": "lib",
-        "rootDir": ".",
+        "rootDir": "src",
         "jsx": "react",
-        "allowSyntheticDefaultImports": true,
         "noImplicitAny": true,
-        "module": "ESNext",
-        "moduleResolution": "node",
-        "lib": ["es2015", "dom"],
-        "target": "es5",
-        "sourceMap": true
+        "allowSyntheticDefaultImports": true
     },
-    "include": ["./src/**/*", "./test/**/*"],
+    "include": ["./src/**/*"],
     "exclude": ["./src/index.umd.ts"]
```

Comment:
does the `exclude` need to be amended as well?

Reply:
This is just for the commonjs build, the `index.umd.ts` file is the entry-point for the UMD file.